public/lesswrong-editor/src/ckeditor5-math/utils.js

```diff
@@ -55,14 +55,14 @@ async function wait( seconds ) {
 	} );
 }
 
-const MAX_MATHJAX_WAITING_PERIODS = 3;
-const MATHJAX_WAITING_PERIOD_LENGTH = 100;
+const MAX_MATHJAX_WAITING_PERIODS = 10;
+const MATHJAX_WAITING_PERIOD_LENGTH = 300;
```

Comment:
interested in knowing what exactly is going on here

Reply:
We load mathJax via a script tag which takes a bit to load and run. In order to deal with that, we check in a loop whether MathJax has finished loading. Previously we waited 3 times for 100ms each, now we wait 10 times for 300ms each. The previous one sometimes didn't wait long enough, in particular on slower connections.